src/components/AnonymousChat/child/ChatItemTargetText.tsx

```diff
@@ -100,7 +100,7 @@ const ChatItemTargetText = ({
   };
 
   return (
-    <View style={styles.chatContainer}>
+    <View style={[styles.chatContainer, isContinuous ? {marginTop: dimen.normalizeDimen(-4)} : {}]}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional application of `marginTop` based on `isContinuous` is a good approach for adjusting the spacing between continuous chat messages. However, using negative margin (`dimen.normalizeDimen(-4)`) can sometimes lead to overlapping elements or other layout issues. Please make sure this change has been adequately tested in various scenarios to ensure it doesn't cause any visual bugs.
```diff
-    <View style={styles.chatContainer}>
+    <View style={[styles.chatContainer, isContinuous ? {marginTop: dimen.normalizeDimen(-4)} : {}]}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->